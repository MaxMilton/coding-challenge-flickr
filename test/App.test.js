'use strict';

// set up mocks
jest.mock('../src/jsonp.js');

const App = require('../src/App.html');
// const store = require('../src/store.js');

// XXX: Using a shallow clone style of mounting the component would be much
// better but there's nothing out of the box for Svelte. If given more time I
// would create my own mount method with stubs for child components.

describe('App root component', () => {
  it('renders correctly', () => {
    expect.assertions(2);
    const target = document.createElement('div');
    function wrapper() {
      new App({ target });
    }
    expect(target.innerHTML).toMatchSnapshot();
    expect(wrapper).not.toThrow();
  });
});
