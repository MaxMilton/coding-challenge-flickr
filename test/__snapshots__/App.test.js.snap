// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App root component renders correctly 1`] = `""`;
